Website/Providers/DataProviders/SqlDataProvider/07.04.02.SqlDataProvider

```diff
@@ -521,6 +521,174 @@ AS
 	WHERE f.WorkflowID IS NULL
 GO
 
+/* Add New OutputCache Table */
+/*****************************/
+
+IF  NOT EXISTS (SELECT * FROM dbo.sysobjects WHERE id = OBJECT_ID(N'{databaseOwner}{objectQualifier}OutputCache') AND OBJECTPROPERTY(id, N'IsTable') = 1)
+	BEGIN
+		CREATE TABLE {databaseOwner}{objectQualifier}OutputCache
+			(
+				[CacheKey] [varchar](36) NOT NULL,
+				[ItemId] [int] NOT NULL,
```

Comment:
What does ItemID refer to? if it is the TabID or FIleID or ModuleID, you need to add a domain (ItemType) as well, otherwise you would return cache for multiple different objects (the ItemType param would needed to be added as param to the CRUD procedures as well)
In this case, you might need an additional non-unique Index (ItemType, ItemID), and ideally set it as clustered